tests/engine-upgrade.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: Run ovirt-ansible roles on containerized environments
+  hosts: engine
+  vars_files:
+    - passwords.yml
+  vars:
+    ovirt_engine_setup_type: "ovirt-engine"
+    ovirt_engine_setup_version: "{{ ovirt_engine_setup_version }}"
+    ovirt_engine_setup_hostname: "localhost"
+    ovirt_engine_setup_organization: "example.com"
+    ovirt_engine_setup_firewall_manager: null
+    ovirt_engine_setup_dwh_db_configure: false
+    ovirt_repositories_ovirt_release_rpm: "{{ ovirt_release_rpm }}"
+  roles:
+    - role: oVirt.repositories
+    - role: oVirt.engine-setup
+      # --accept-defaults needed because of: https://bugzilla.redhat.com/show_bug.cgi?id=1518697
```

Comment:
Would you be interested in using https://bugzilla.redhat.com/show_bug.cgi?id=1396925 instead? I set needinfo on Lukas but got no reply... Current project may be the best way to move this forward.

Reply:
I 'll add it as well, sure.

Reply:
I agree with having otopi answer files only. However we need to wait for the patches to be merged and available. i would say it is out of scope of this patch, and I would rather handle this after this and mentioned BZ patch is released.

Reply:
Merged the patch earlier this morning, should be available later on today - in master-snapshot repo.